sqldelight-gradle-plugin/build.gradle

```diff
@@ -141,6 +157,11 @@ tasks.named("shadowJar") {
   archiveClassifier.set("")
   configurations = [project.configurations.shade]
 
+  minimize {
+    exclude(dependency(libs.intellij.testFramework.get()))
+    exclude(dependency(libs.intellij.langImpl.get()))
+  }
```

Comment:
oh nice :+1:

Reply:
Unlike R8/Proguard Shadow only support dependency notation, not classes. But at the other hand, it is an advantage for dialect authors able to use the whole package without changing some `keep` rules and waiting for the next release of sqldelight.